aten/src/ATen/native/sparse/SparseTensor.cpp

```diff
@@ -204,7 +204,7 @@ SparseTensor new_with_tensor_and_size_sparse(const LongTensor& indices, const Te
 
 SparseTensor clone_sparse(const SparseTensor& self) {
   SparseTensor other = new_with_dims_and_size_sparse(self.type(), self._sparseDims(), self._denseDims(), self.sizes());
-  _copy_into_sparse(other, _get_sparse_impl(self)->indices(), _get_sparse_impl(self)->values());
+  _copy_into_sparse(other, _get_sparse_impl(self)->indices(), _get_sparse_impl(self)->values(), false);
```

Comment:
So if i understand correctly, for CUDA types, it is okay if `.clone()` is async (and in fact is preferred); for CPU types, there is no such thing as async copy. And `clone()` doesn't do cross device copy.... maybe we should just set to `true` here? Correct me if I'm wrong though.

Reply:
Yeah, sure, I agree that set `non_blocking=true` here might make more sense. So IIUC, `non_blocking=true` should not cause any issue when:
```
x = SparseTensor.clone()
y = x.clone()
```
Is it correct?